src/hooks/useLayoutDependentValue.ts

```diff
@@ -2,12 +2,14 @@ import { useEffect, useRef, useState } from "react";
 
 // Returns the result of f, computed on the first call and recomputed whenever the window changes size.
 export function useLayoutDependentValue<T>(f: () => T): T {
-  const [state, setState] = useState<T>(f());
+  const [state, setState] = useState<T>(() => f());
```

Comment:
This change is needed because calling a function in useState results in the function being evaluated every hook, which is not what we want.

Reply:
My bad—thank you!